VideoLocker/res/drawable/edx_segmented_control_left_background.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shape="rectangle">
+    <solid android:color="@color/edx_segmented_control_background_selector" />
+    <stroke
+        android:width="@dimen/edx_hairline"
+        android:color="@color/edx_grayscale_neutral_light" />
+    <corners
+        android:bottomLeftRadius="@dimen/edx_box_radius"
+        android:bottomRightRadius="0dp"
+        android:radius="@dimen/edx_box_radius"
+        android:topLeftRadius="@dimen/edx_box_radius"
+        android:topRightRadius="0dp" />
```

Comment:
Hmm, since we can't define relative directions (start, end) in drawables, and I don't think `GradientDrawable` supports auto-mirroring either, we might have to define inverted versions of these in the `drawable-ldrtl` directory in order to support RTL mode (where the segment positioning will be inverted)...